src/app/login/login-form.component.ts

```diff
@@ -51,30 +51,30 @@ export class LoginFormComponent {
 
   createUser({ name, password, repeatPassword }: {name: string, password: string, repeatPassword: string}) {
     if (password === repeatPassword) {
-      this.couchService.put('_users/org.couchdb.user:' + name,
-      { 'name': name, 'password': password, 'roles': [], 'type': 'user', 'isUserAdmin': false })
+      this.couchService.put('_users/org.couchdb.user:' + name.toLowerCase(),
+      { 'name': name.toLowerCase(), 'password': password, 'roles': [], 'type': 'user', 'isUserAdmin': false })
         .subscribe((data) => {
-          this.message = 'User created: ' + data.id.replace('org.couchdb.user:', '');
+          this.planetMessageService.showMessage('User created: ' + data.id.replace('org.couchdb.user:', ''));
           this.welcomeNotification(data.id);
           this.login(this.model, true);
-        }, (error) => this.message = '');
+        }, (error) => this.planetMessageService.showMessage(error.error.reason));

```

Comment:
Let's replace this message with something user friendly.  `There was an error in creating a new user` would be better